clusterloader2/docs/GETTING_STARTED.md

```diff
@@ -0,0 +1,303 @@
+Also, we will need to specify template for our deployment, which we will do later.
+For now, let's assume that this template allows us to specify number of replicas in deployment.
+```yaml
+- name: Create deployment
+  phases:
+  - namespaceRange:
+      min: 1
+      max: 1
+    replicasPerNamespace: 1
+    tuningSet: Uniform1qps
+    objectBundle:
+    - basename: test-deployment
+      objectTemplatePath: "deployment.yaml"
+      templateFillMap:
+        Replicas: 10
+```
+Now, we need to wait for pods in this deployment to be in Running state before we measure PodStartupLatency.
+Let's create next step that will do it:
+```yaml
+- name: Wait for pods to be running
+  measurements:
+  - Identifier: WaitForControlledPodsRunning
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: gather
+```
+Now we can measure PodStartupLatency in next step:
+```yaml
+- name: Measure pod startup latency
+  - Identifier: PodStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: gather
+```
+Whole `config.yaml` will look like this:
+```yaml
+name: test
+
+namespace:
+  number: 1
+
+tuningSets:
+- name: Uniform1qps
+  qpsLoad:
+    qps: 1
+
+steps:
+- name: Start measurements
+  measurements:
+  - Identifier: PodStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: start
+      labelSelector: group = test-pod
+      threshold: 20s
+  - Identifier: WaitForControlledPodsRunning
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: start
+      apiVersion: apps/v1
+      kind: Deployment
+      labelSelector: group = test-pod
+      operationTimeout: 120s
+- name: Create deployment
+  phases:
+  - namespaceRange:
+      min: 1
+      max: 1
+    replicasPerNamespace: 1
+    tuningSet: Uniform1qps
+    objectBundle:
+    - basename: test-deployment
+      objectTemplatePath: "deployment.yaml"
+      templateFillMap:
+        Replicas: 10
+- name: Wait for pods to be running
+  measurements:
+  - Identifier: WaitForControlledPodsRunning
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: gather
+- name: Measure pod startup latency
+  - Identifier: PodStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: gather
+```
+
+By default, clusterloader will delete auto-created namespaces so we don't need to worry with cleaning up cluster.
+
+Now, in order to finish our first test, we need to specify deployment template.
+You can think of it as regular kubernetes object, but with templating.
+CL2 by default adds parameter `Name` that you can use in your template.
+In our config, we also passed `Replicas` parameter.
+We need to remember to set correct labels so PodStartupLatency and WaitForControlledPodsRunning will watch correct pods.
+So our template for deployment will look like this (`deployment.yaml` file):
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Name}}
+  labels:
+    group: test-pod
```

Comment:
"test-pod" as a label for Deployment is really misleading...

Reply:
Changed to test-deployment.